src/api/routes/index.js

```diff
@@ -24,4 +26,6 @@ router.use('/notifications', notificationRouter);
 router.use('/termsAndConditions', termsAndConditionsRouter);
 router.use('/comment', reactionsRouter);
 
+router.use('/', categoryRouter);
```

Comment:
can you give this route a name? eg: `/categories`

Reply:
I agree with @alainburindi @truestbyheart